specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json

```diff
@@ -2045,6 +2045,35 @@
             "name": "Capabilities",
             "modelAsString": true
           }
+        },
+        "locations": {
```

Comment:
how is the location of the primary different from location of the resource as provided via the top level property?

Reply:
Location provided via top level property is immutable object referring to the original location (where resource is created).
while this primary location refers to the current location of the resource which can be different post fail over.